qucs-doc/tutorial/textmode/content.tex

```diff
@@ -151,7 +151,7 @@
 
 \begin{figure}[h]
 \begin{center}
-  \includegraphics[angle=0,width=0.8\linewidth]{rc_ac_python.eps}
```

Comment:
> for some reason the change got lost 

indeed. i dont know what happened. but more changes are lost (such as 69a8725d0cc90f8e5c4f9a6e97687e745d2dd7cd)

sorry about that. will try to sort this out!